src/azure-cli/azure/cli/command_modules/acs/azuremonitormetrics/azuremonitorprofile.py

```diff
@@ -50,6 +51,9 @@ def link_azure_monitor_profile_artifacts(
 
 # pylint: disable=line-too-long
 def unlink_azure_monitor_profile_artifacts(cmd, cluster_subscription, cluster_resource_group_name, cluster_name):
+    msg = "Executing this action will result in the removal of all custom resources for the `azmonitoring.coreos.com` custom resource defintion. Are you certain that you wish to proceed with this operation?"
+    if not prompt_y_n(msg, default="n"):
+        raise ClientError("Operation cancelled by user")
```

Comment:
May I ask will this change cause a breaking change to the customers' automation scripts?

Reply:
This looks like a breaking change. Can you queue some related live tests to see if there are any errors?

Reply:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=90453125&view=results). I want to warn users that if they disable the addon then the custom resources will get deleted. What is the process to allow this type of breaking change to be done?

Reply:
I have updated the code to only add the warning and not ask for a prompt as this wasn't a hard requirment. Queued live tests again : [here](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=90454465&view=results)